### DISCUSSION TITLE
BdB Avatar Community Marketplace,  built on @Rarible Protocol.

### DISCUSSION THREAD

**Discussion by efeblissai** [community/community#29154] [Other Feature Feedback, Questions, & Ideas]
![Screenshot_20220809-231654_Gallery.jpg](https://user-images.githubusercontent.com/62979725/184138373-d5ab0f21-e367-43c1-9eda-d08a9f813d7c.jpg)https://bdbavatars.wlbl.xyz/explore/ETHEREUM:0xd6dc71f64ee75ab5108302a236

**Comment by augustoaraujoo** [comment]
nice project

**Reply by efeblissai** [reply]
[](https://bdbavatars.wlbl.xyz/[](url))
